plugins/UM3NetworkPrinting/src/UM3OutputDevicePlugin.py

```diff
@@ -70,6 +64,18 @@ def __init__(self):
         self._service_changed_request_queue = Queue()
         self._service_changed_request_event = Event()
         self._service_changed_request_thread = Thread(target=self._handleOnServiceChangedRequests, daemon=True)
+
+        from cura.CuraApplication import CuraApplication
+        CuraApplication.getInstance().initializationFinished.connect(self._initialize)
+
+    def _initialize(self) -> None:
```

Comment:
Please document this. It makes no sense for the reader of this code that this has to be delayed to after `initializationFinished`. It might also be more useful to call this function `_onInitializationFinished` instead of a meaningless `_initialize`.